api/queries_pr.go

```diff
@@ -629,17 +631,55 @@ func CreatePullRequest(client *Client, repo *Repository, params map[string]inter
 	return pr, nil
 }
 
-func UpdatePullRequestReviews(client *Client, repo ghrepo.Interface, params githubv4.RequestReviewsInput) error {
-	var mutation struct {
-		RequestReviews struct {
-			PullRequest struct {
-				ID string
-			}
-		} `graphql:"requestReviews(input: $input)"`
+// AddPullRequestReviews updates the requested reviewers on a pull request using the REST API.
```

Comment:
How about highlighting the fact that this is add operation (as opposed to set/put)?


```suggestion
// AddPullRequestReviews adds the given users/teams to requested reviewers on a pull request using the REST API.
```